tests/test_stumped_two_subsequences_nan_inf_A_B_join.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import numpy.testing as npt
+import pandas as pd
+from stumpy import stumped, core
+from dask.distributed import Client, LocalCluster
+import pytest
+import warnings
+import utils
+
+
+@pytest.fixture(scope="module")
+def dask_client():
+    cluster = LocalCluster(n_workers=2, threads_per_worker=2)
+    client = Client(cluster)
+    yield client
+    # teardown
+    client.close()
+    cluster.close()
+
+
+test_data = [
+    (
+        np.array([9, 8100, -60, 7], dtype=np.float64),
+        np.array([584, -11, 23, 79, 1001, 0, -19], dtype=np.float64),
+    ),
+    (
+        np.random.uniform(-1000, 1000, [8]).astype(np.float64),
+        np.random.uniform(-1000, 1000, [64]).astype(np.float64),
+    ),
+]
+
+substitution_locations = [(0, 1), (-1, slice(1, 3)), (slice(1, 3), 1), ([0, 3], 1)]
+
+
+@pytest.mark.filterwarnings("ignore:numpy.dtype size changed")
+@pytest.mark.filterwarnings("ignore:numpy.ufunc size changed")
+@pytest.mark.filterwarnings("ignore:numpy.ndarray size changed")
+@pytest.mark.filterwarnings("ignore:\\s+Port 8787 is already in use:UserWarning")
+@pytest.mark.parametrize("T_A, T_B", test_data)
+@pytest.mark.parametrize(
+    "substitution_location_A, substitution_location_B", substitution_locations
+)
+def test_stumped_two_subsequences_nan_inf_A_B_join(
```

Comment:
In this particular case, would you mind adding an additional test where the`T_A_sub` is `np.nan` and `T_B_sub` is `np.inf`? Basically, I'm just swapping "where" `np.nan`/`np.inf` exists

Reply:
One single test or all eight tests again, just reversed?